lib/autoyast.rb

```diff
@@ -278,14 +284,38 @@ def apply_extracted_files(scope)
 
     @system_description[scope].files.map do |file|
       if file.user
-        @chroot_scripts << "chown #{file.user}:#{file.group} '#{File.join("/mnt", file.name)}'"
+        @chroot_scripts << "chown --no-dereference #{file.user}:#{file.group} '#{File.join("/mnt", file.name)}'"
       end
-      if file.mode
+      if file.mode && !file.link?
         @chroot_scripts << "chmod #{file.mode} '#{File.join("/mnt", file.name)}'"
       end
     end
   end
 
+  def apply_changed_managed_files
```

Comment:
Assignment Branch Condition size for apply_changed_managed_files is too high. [51.28/15]